projects/core/src/lib/components/header/header.component.scss

```diff
@@ -0,0 +1,165 @@
+@import "../../../../../../src/assets/styles/mixins";
+@import "../../../../../../src/assets/styles/utils";
+
+.jsmu-header {
+  background-color: rgb(38 39 52);
+  height: 70px;
+  padding: .3vh 1vw;
+  font-size: 1.4vw;
+  z-index: 1;
+  border-bottom: 1px rgba(54, 55, 51, 0.8) solid;
+
+  &__list-item {
+    list-style: none;
+    border-right: 3px rgba(6, 6, 18, 0.89) dotted;
+    padding-right: 1vw;
+
+    :hover {
+      color: $lightBlueColor;
+    }
+  }
+
+  &__list-item:last-child {
```

Comment:
please make nesting for :last-child  into  &__list-item

Reply:
done